game-core/src/main/java/games/strategy/engine/framework/startup/ui/posted/game/pbf/ForumPosterEditorViewModel.java

```diff
@@ -120,7 +120,10 @@ synchronized void setTopicId(final String topicId) {
   }
 
   synchronized boolean areFieldsValid() {
-    return isTopicIdValid() && isForumUsernameValid() && isForumPasswordValid();
+    return isTopicIdValid()
+        && ((isForumUsernameValid() && isForumPasswordValid())
+            || ClientSetting.aaForumToken.isSet()
+            || ClientSetting.tripleaForumToken.isSet());
```

Comment:
This class is just band-aided so I could use the "Test PbF" button and everything compiles properly.
This will obviously change in the final verison of this PR, but I haven't had them time for everything today.